.github/workflows/test_vectors.yml

```diff
@@ -24,6 +24,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Install gnome-keyring
+        run: sudo apt-get install gnome-keyring
```

Comment:
What is this needed for?

Reply:
One of the python dependencies refuses to build without it; that was the previous build failure.